src/interfaces/design.d.ts

```diff
@@ -6,6 +6,14 @@ export interface IDesign {
   courtSize: ICourtSize;
 }
 
+export interface IDesignDetailed extends IDesign {

```

Comment:
Is "detailed" should be "detail"? :)

Reply:
Thanks Tim! "detail" sounds make more sense - have already renamed it :)